torch_xla/csrc/tensor.cpp

```diff
@@ -1575,6 +1584,17 @@ XLATensor XLATensor::tril(const XLATensor& input, xla::int64 diagonal) {
       ir::MakeNode<ir::ops::Tril>(input.GetIrValue(), diagonal));
 }
 
+XLATensor XLATensor::trace(const XLATensor& input) {
```

Comment:
Given this is simple, I think we can leave as is. But norm() needs generic trace.

Reply:
Yes, I know, I didn't realize this isn't good enough for `norm` until after I implemented it, but there's no harm in keeping it anyway.